<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditContatos</class>
 <widget class="QDialog" name="EditContatos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="idContato">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Id</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>391</width>
     <height>124</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nome:    </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editNome"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Telefone:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editTelefone"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Email:     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editEmail"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="btnExcluirContato">
        <property name="text">
         <string>Excluir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEditarContato">
        <property name="text">
         <string>Editar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
